src/main/java/play/dev/filewatch/DefaultFileWatchService.java

```diff
@@ -38,7 +37,7 @@ public DefaultFileWatchService(LoggerProxy logger, boolean isMac) {
   }
 
   @Override
-  public FileWatcher watch(Iterable<File> filesToWatch, Supplier<Void> onChange) {
+  public FileWatcher watch(Iterable<File> filesToWatch, Runnable onChange) {
```

Comment:
IMHO we should use just Runnable instead of `Supplier<Void>`, because using `Supplier<Void>` has absolutely no advantage, but instead we always, always have to return null. Does not make sense. Just normal runnable is much nicer IMHO.